modules/olm-disconnected-operatorhub-disconnected-mode.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-disconnected.adoc
+
+[id="olm-disconnected-operatohub-disconnected-mode_{context}"]
+= Configuring OperatorHub in disconnected mode
+
+Cluster administrators can configure OLM and OperatorHub to use local content in
+a disconnected mode.
+
+.Prerequisites
+
+* Cluster administrator access to a connected or disconnected {product-title} cluster and its internal registry.
+* Separate workstation with full Internet access.
+* If pushing images to the {product-title} cluster's internal registry, the registry must be exposed with a route.
+* `podman` version 1.5.1+
+
+.Procedure
+
+. **Disable the default OperatorSources.**
+
+.. Edit the `OperatorHub` configuration resource:
++
+----
+$ oc edit OperatorHub cluster
+----
+
+.. Add `disableAllDefaultSources: true` to the spec:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: OperatorHub
+spec:
+  disableAllDefaultSources: true
```

Comment:
Nit, you can set this with a copy/pastable command like:

```console
$ oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
```

to avoid dropping into an editor.

Reply:
:+1: Updated.